src/POGOProtos/Networking/Signature.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+
+message Signature {
+
+    message LocationFix {
```

Comment:
Curious, why the name Location _Fix_? Because it presumably overrides the location information already present in the wrapper?